README.md

```diff
@@ -1,49 +1,103 @@
 # migrator
 
-Command line interface (CLI) to migrate a computer to run balenaOS.
+Command line interface (CLI) to automate migration of a computer to run balenaOS.
 
 **WARNING This tool will overwrite the operating system and all data on the computer that runs it.** Be sure to save any important data before using the migrator.
 
+Presently supports migration only from a computer running Windows 10 on UEFI based firmware, with Ethernet DHCP (self-configured) networking.
+
 ## About
 
-The migrator provides a command prompt / terminal window to run etcher SDK's [migrator function](https://github.com/balena-io-modules/etcher-sdk/tree/master/lib/migrator). The migrator is an [open-source project on
-GitHub](https://github.com/balena-os/migrator/), and your contributions are welcome!
+Migrator accepts a balenaOS image file for your computer, formats it onto  the computer's disk/flash storage, and then replaces the current operating system with balenaOS derived from that image. As a CLI application, it's easy to automate, and you can run it remotely.
 
-Presently supports migration only from a computer running Windows 10 on UEFI based firmware.
+*Our goal is to make it easy to migrate a computer to use balenaOS.* Let us know how it works for you. As an open source project, we're also happy to accept code and documentation contributions. See *Development* below.
 
-## Installation and Building
+## Getting Started
 
-Clone the migrator repository, install the node modules and build it as shown below.
+### Prepare image
+Migrator expects a pre-configured image for your device. You only need to perform these download and configure steps once for a fleet.
 
+Use [balenaCLI](https://docs.balena.io/reference/balena-cli/#os-download-type) or the balena [web site](https://www.balena.io/os#download-os) to download the image appropriate for your computer. Then configure it for your fleet. If you download from the web site, first unzip the compressed file.
```

Comment:
We need to specify a flasher image. Eventually we will most likely unify flasher and non flasher, but right now it's only supported for flasher images.

Reply:
How can a user determine if their device accepts a flasher image? Can we just say that the device must boot from internal storage?

Reply:
As you know only x86_64 can run Windows, and both Generic GPT and MBR images are flasher images, so for the Windows migrator all images are flasher images.
When Linux come into play, we will need to deploy flasher images for all devices, or unify the flasher/non-flasher, whatever comes sooner.

Reply:
Thanks for clarifying. According to [this doc](https://learn.microsoft.com/en-us/windows/arm/overview), Windows now runs on ARM as well.

Reply:
I'm looking at the supported device types, and I'd like to be precise on which images are flasher images. Is it only Generic x86_64 (GPT) and Generic x86_64 (legacy MBR)? For example, is Intel NUC a flasher image? A user of that device type might expect to use that specific image.